doc/lsst.meas.extensions.shapeHSM/hsm-shape.rst

```diff
@@ -0,0 +1,42 @@
+.. py:currentmodule:: lsst.meas.extensions.shapeHSM
+
+PSF-correction algorithms
+=========================
+
+The purpose of the `HsmShape` algorithms is to perform PSF correction to
+measure galaxy shapes for the purpose of weak lensing.  The plugins can do
+several methods of PSF correction, described and tested on simulated
+data in Hirata & Seljak (2003),
+http://adsabs.harvard.edu/abs/2003MNRAS.343..459H.
+
+The PSF correction methods are
+
+1. KSB --
```

Comment:
It would be nice to have each of these abbreviations be a link to the plugin name/API. They're all implemented as trivial subclass wrappers, but at least this way there's an explicit name for someone to look up.

I think something like this?

```suggestion
1. :py:class:`KSB <lsst.meas.extensions.shapeHSM.HsmShapeKsbPlugin>` --
```